app-quizz/src/app/components/Button/button.tsx

```diff
@@ -1,11 +1,17 @@
 type ButtonProps = {
-    title: string
-} 
+  title: string;
+  function: () => void;
+};
 
 const Button = (props: ButtonProps) => {
-    return <button className="bg-teal-900 hover:bg-teal-800 text-white font-bold py-2 px-4 rounded">
-            {props.title}
-        </button>
-}
+  return (
+    <button
+      className="bg-teal-900 hover:bg-teal-800 text-white font-bold py-2 px-4 rounded"
```

Comment:
S: maybe we can receive a `class` string type here for doing more general this button